// @strapi/upload/dist/admin/future/App.mjs
import { jsxs, jsx } from 'react/jsx-runtime';
import 'react';
import { Page } from '@strapi/admin/strapi-admin';
import { useIntl } from 'react-intl';
import { Routes, Route } from 'react-router-dom';
import 'byte-size';
import 'date-fns';
import { getTrad } from '../utils/getTrad.mjs';
import 'qs';
import '../utils/typeFromMime.mjs';
import '../utils/urlYupSchema.mjs';
import { AIGenerationPage } from './pages/AIGenerationPage.mjs';
import { MediaLibraryPage } from './pages/MediaLibraryPage.mjs';

const UnstableMediaLibraryPage = ()=>{
    const { formatMessage } = useIntl();
    const title = formatMessage({
        id: getTrad('plugin.name'),
        defaultMessage: 'Media Library'
    });
    return /*#__PURE__*/ jsxs(Page.Main, {
        children: [
            /*#__PURE__*/ jsx(Page.Title, {
                children: title
            }),
            /*#__PURE__*/ jsxs(Routes, {
                children: [
                    /*#__PURE__*/ jsx(Route, {
                        index: true,
                        element: /*#__PURE__*/ jsx(MediaLibraryPage, {})
                    }),
                    /*#__PURE__*/ jsx(Route, {
                        path: "ai-generation",
                        element: /*#__PURE__*/ jsx(AIGenerationPage, {})
                    })
                ]
            })
        ]
    });
};

export { UnstableMediaLibraryPage };
