// @strapi/upload/dist/admin/future/pages/MediaLibraryPage.mjs
import { jsxs, jsx } from 'react/jsx-runtime';
import 'react';
import { Layouts, SearchInput } from '@strapi/admin/strapi-admin';
import { Box, Flex } from '@strapi/design-system';
import { useIntl } from 'react-intl';
import 'byte-size';
import 'date-fns';
import { getTrad } from '../../utils/getTrad.mjs';
import 'qs';
import '../../utils/typeFromMime.mjs';
import '../../utils/urlYupSchema.mjs';

const MediaLibraryPage = ()=>{
    const { formatMessage } = useIntl();
    return(/**
     * NOTE:
     *
     * The design differs from our current Layouts component.
     * Either we find a way to make it work with our current Layouts component
     * or we will have to write our own custom layout.
     */ /*#__PURE__*/ jsxs(Layouts.Root, {
        children: [
            /*#__PURE__*/ jsx(Layouts.Header, {
                navigationAction: /*#__PURE__*/ jsx(Box, {
                    children: "TODO: Breadcrumbs"
                }),
                title: "TODO: Folder location",
                primaryAction: /*#__PURE__*/ jsxs(Flex, {
                    gap: 2,
                    children: [
                        /*#__PURE__*/ jsx(SearchInput, {
                            label: formatMessage({
                                id: getTrad('search.label'),
                                defaultMessage: 'Search for an asset'
                            }),
                            trackedEvent: "didSearchMediaLibraryElements",
                            trackedEventDetails: {
                                location: 'upload'
                            }
                        }),
                        "TODO: Toolbar"
                    ]
                })
            }),
            /*#__PURE__*/ jsx(Layouts.Content, {
                children: "TODO: List/Grid views"
            })
        ]
    }));
};

export { MediaLibraryPage };
